actions/src/pull_requests/labelCheck.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
Should we make sure that the label cannot be removed by a person and only can be removed once there aren't any merge conflicts?

Reply:
There are multiple don't merge labels. Should this be done only for the `PR don't merge: merge conflict` label?

Reply:
Hmm, not sure, we can probably do it in a different PR thought.

Reply:
Yes.